cmake/nuttx/px4_impl_nuttx.cmake

```diff
@@ -290,35 +290,35 @@ function(px4_nuttx_add_export)
 		set(config_nuttx_hw_stack_check_${CONFIG} y CACHE INTERNAL "" FORCE)
 	endif()
 
-	# export
+	# configure
 	file(GLOB_RECURSE config_files ${PX4_SOURCE_DIR}/nuttx-configs/${CONFIG}/*)
-
-	add_custom_command(OUTPUT ${PX4_BINARY_DIR}/nuttx_export_${CONFIG}.stamp
-		#COMMAND ${ECHO} Configuring NuttX for ${CONFIG} with ${config_nuttx_config}
-		#COMMAND ${MAKE} --no-print-directory -C${nuttx_src}/nuttx -r --quiet distclean
+	add_custom_command(OUTPUT ${nuttx_src}/nuttx/.config
 		COMMAND ${CP} -rp ${PX4_SOURCE_DIR}/nuttx-configs/*.mk ${nuttx_src}/nuttx/
 		COMMAND ${CP} -rp ${PX4_SOURCE_DIR}/nuttx-configs/${CONFIG} ${nuttx_src}/nuttx/configs
-		COMMAND cd ${nuttx_src}/nuttx/tools && ./configure.sh ${CONFIG}/${config_nuttx_config} && cd ..
-		#COMMAND ${ECHO} Exporting NuttX for ${CONFIG}
+		COMMAND cd ${nuttx_src}/nuttx/tools && ./configure.sh ${CONFIG}/${config_nuttx_config}
+		DEPENDS ${DEPENDS} nuttx_patch_${CONFIG} ${config_files}
+		WORKING_DIRECTORY ${PX4_BINARY_DIR}
+		COMMENT "Configuring NuttX for ${CONFIG} with ${config_nuttx_config}")
+
+	# manual reconfigure helper
+	add_custom_target(reconfigure_nuttx_${CONFIG}
```

Comment:
The first is fine. - The thing is really not working 100% because the boards were removed from upstream and do not have an appearance in Kconfig.  So for now we should just leave it as in and I can futz with it later.